pkgs/development/embedded/fpga/trellis/default.nix

```diff
@@ -1,30 +1,27 @@
-{ lib, stdenv, fetchFromGitHub
```

Comment:
This entry arglist apparently just gets reformatted anyway when people format the codebase, so don't bother.

Reply:
I'll keep this in mind for the next time.